### PR TITLE
implement builtin math function floor_int and floor_real

### PR DESCRIPTION
**Pull request by JinheLin**

### CODE REVIEW

File: src/coprocessor/select/xeval/builtin_math.rs [hunk 1]
```diff
@@ -15,7 +15,7 @@ use tipb::expression::Expr;
 use coprocessor::codec::datum::Datum;
 use super::{Evaluator, EvalContext, Result, Error};
 
-const ERROR_UNIMPLEMENTED: &'static str = "unimplemented";
+const _ERROR_UNIMPLEMENTED: &'static str = "unimplemented";
```
Comment (JinheLin) [inline src/coprocessor/select/xeval/builtin_math.rs]: Because it did not use and would result in compile error. 

Maybe it can be deleted.

Comment (siddontang) [inline src/coprocessor/select/xeval/builtin_math.rs]: /cc @AndreMouche

**Comment by JinheLin** [comment]
image

There seems to be something wrong with Travis CI.

**Comment by JinheLin** [comment]
OK @AndreMouche

**Pull request closed without merging**